README.md

```diff
@@ -52,6 +52,45 @@ There are 4 types of modules and 5 layouts currently supported.
   pathfora.initializeWidgets([module]);
   ```
 
+## Communication
+**`slack`** - There’s a [slack channel](https://join.slack.com/t/pathforajs/shared_invite/enQtMjcyNDEzMDY4NzIxLTI5ZDIyMDI2NGEzNjU4NDE3MTgyOWQ2YzM5MzhjZjVmZDljMDE3NmU5MDFmYmExNTA5ODlhZmE4NmM2ZmE3MTI). Feel free to join and collaborate!
+
+## Contributing to Pathfora
```

Comment:
I think this is a nicely written doc on contribution, but perhaps deserving of its own markdown file. We could make a `contributing.md` file in the repo and then link it from the README. I feel like the readme's getting a bit long.

Reply:
Ahhh yeah, kind of like what we do with the License section for the other PR. I wholeheartedly agree with this! Will make the change and move it on over.

Reply:
@sarahyablok Updated based on your suggestion.